src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/CodePointFilter.cs

```diff
@@ -49,46 +50,46 @@ public CodePointFilter(params UnicodeRange[] allowedRanges)
         }
 
         /// <summary>
-        /// Allows the character specified by <paramref name="c"/> through the filter.
+        /// Allows the character specified by <paramref name="character"/> through the filter.
         /// </summary>
         /// <returns>
         /// The 'this' instance.
         /// </returns>
-        public CodePointFilter AllowChar(char c)
+        public CodePointFilter AllowChar(char character)
         {
-            _allowedCharsBitmap.AllowCharacter(c);
+            _allowedCharsBitmap.AllowCharacter(character);
             return this;
         }
 
         /// <summary>
-        /// Allows all characters specified by <paramref name="chars"/> through the filter.
+        /// Allows all characters specified by <paramref name="characters"/> through the filter.
         /// </summary>
         /// <returns>
         /// The 'this' instance.
         /// </returns>
-        public CodePointFilter AllowChars(params char[] chars)
+        public CodePointFilter AllowChars(params char[] characters)
         {
-            if (chars != null)
+            if (characters != null)
```

Comment:
Ultimately I think the bigger issue is that the public entry points to the various encoding-related APIs are not fully consistent:

1. Some do null checks and no-op (this method)
2. Some do null checks and throw (various other APIs, such as `TextEncoderExtensions.Encode(...)`)
3. Some don't do null checks and NullRef (such as `AllowChars(string)` right below this method).

I agree that between 1. and 2. it's debatable and really depends on the scenario, but even within the scope of this change I think it might be inconsistent even for particular scenarios. E.g. what's so wrong about a `null` value for a `value` parameter? Sometimes you're getting the value from "some other place" that might return null when there's no data, but you don't care, you just want it to be safe to render out as HTML/JS/URL so you just want to get back `""` or whatever. For some methods it totally makes sense to throw: e.g. you called an API that requires passing in a particular encoding, and you passed in `null` - the operation can't possibly succeed, and it's clearly a bug in the app's code.